### DISCUSSION TITLE
runLocally running relative to recipe directory

### DISCUSSION THREAD

**Discussion by lfolco** [deployphp/deployer#2935] [Help needed]
In v6, I used to be able to specify a path for `runLocally` along these lines:

```
runLocally('{{bin/php}} {{current_path}}/bin/magento dev:urn-catalog:generate .idea/misc.xml');
```

With v7, I'm getting the error that it cannot find `bin/magento`. So how do I tell `runLocally` to use the current path (or the release path, etc.) without having to deal with the current directory where the recipe lives?

The docs say something about setting an environment variable:

```
DEPLOYER_ROOT=. vendor/bin/dep taskname`
```

But this implies that I need to specify it for each task?

**Comment by antonmedv** [comment]
What is {{current_path}} stands for? I think it's related to remote host, not localhost. 

Probably specify correct path from deploy.php file

**Reply by lfolco** [reply]
Sorry, should have used `release_or_current_path`

**Reply by lfolco** [reply]
Okay, sorry, I figured it out, I was getting confused about using localhost targets.
